app/jobs/handover_follow_up_job.rb

```diff
@@ -0,0 +1,32 @@
+class HandoverFollowUpJob < ApplicationJob
+  queue_as :default
+
+  def perform(date)
+    ldus = LocalDivisionalUnit.all.where.not(email_address: nil)
+
+    ldus.each do |ldu|
+      offenders = ldu.teams.map { |team| team.case_information.where(com_name: nil).map(&:nomis_offender_id) }.flatten
+                  .map { |off_id| OffenderService.get_offender(off_id) }
```

Comment:
The fallout from POM-486 showed that this isn't enough. We need to make sure that offender is sentenced, and that get_offender doesn't return nil